src/app/shared-components/student-list/student-list.component.ts

```diff
@@ -18,6 +18,7 @@ export class StudentListComponent implements OnInit {
   isStudentsLoaded$: Observable<boolean> | undefined ;
   hasLoadFailed$: Observable<boolean> | undefined ;
   error$: Observable<CustomError | null> | undefined;
+  @Input() isInAdminOrManger: boolean | undefined;
```

Comment:
```suggestion
  @Input() isManagerView: boolean | undefined;
```
Naming is a bit unclear, so if it's true, does it make admin or manager? Try renaming to this :)